web/src/beta/lib/reearth-ui/components/CodeInput/index.tsx

```
@@ -95,6 +97,7 @@ export const CodeInput: FC<CodeInputProps> = ({
         height={height}
         onChange={handleChange}
         onMount={handleEditorDidMount}
+        loading={<>{t("Loading...")}</>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)